src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -127,91 +126,93 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
                     }
                 }
                 String entry = current.toString();
+                if (entry.contains("-")) {
```

Comment:
This would also mean you don't need to modify as much code as you did now because then you don't have the if-clause around all the code.